ShenkinStore/Models/User.cs

```diff
@@ -12,22 +13,32 @@ public class User
         public int UserId { get; set; }
         [Display(Name = "User Name")]
         [DataType(DataType.Text)]
-        [Required(ErrorMessage = "Please enter name"), MaxLength(20)]
+        //[Required(ErrorMessage = "Please enter name"), MaxLength(20)]
```

Comment:
Remove unnecessary comments in this file and separate the parameters as stated in LoginModel.cs comment